app/graphql/types/tag_type.rb

```diff
@@ -2,7 +2,7 @@ module Types
   class TagType < Types::BaseObject
     field :id, ID, null: false
     field :name, String, null: true
-    field :taggings_count, Integer, null: true
+    field_with_deprecated_sibling :taggings_count, Integer, null: true
```

Comment:
maybe putting this below `field :kind`, would result in a better readability with quicker overview of fields and fields with deprecated siblings?